docs-samples/V4/JS/manage-conversation-and-user-state/memory-storage.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Botbuilder v4 SDK - Add suggested actions
+ * 
+ * This bot demonstrates how to manage a conversation state and user state with MemoryStorage.
```

Comment:
Call out that this storage is for prototype only and is not intended for production use. Be sure to set storage to a database like AzureTable storage before publishing your bot.